coordinator/src/test/scala/filodb.coordinator/queryplanner/PlannerHierarchySpec.scala

```diff
@@ -596,272 +588,251 @@ class PlannerHierarchySpec extends AnyFunSpec with Matchers with PlanValidationS
 
   it("should not pushdown root scalar operation into a RemoteExec's promql when query spans multiple partitions") {
     val queryExpectedPairs = Seq(
-      // Instant selector without namespace regex.
```

Comment:
I'll leave these comments out -- I wrote a small piece of code to update/print these tests for me, and comments weren't included in the output. They don't add much value.

I think sometime soon I'll advocate we add self-updating tests to filodb. These things are a huge pain to update, and they disincentivize going wild with promql tests that would otherwise make it ultra-clear in a PR how behavior will change. (Also long sequences of tests like this are a problem I created that everyone else needs to deal with, now.)

Anyways, </rant>, this is a discussion for another time.